### DISCUSSION TITLE
[Feature] Create Portable Apps

### DISCUSSION THREAD

**Discussion by greenfox1505** [ScoopInstaller/Scoop#5450] [Help and Questions]
## Feature Request

A "Scoop For Portable" installer. If this is already possible, I would like the documentation for this easier to find.

#### Is your feature request related to a problem? Please describe.

I am working on a tool that lets you compile a C++ project with a number of optional flags and modules. The intended user of this project is people who would like access to those flags and modules, but don't want to learn the compile chain to do that. The goal is to lower the barrier of entry of a number of features.

Currently the prerequisite to compile this project is `scoop install gcc python scons make`. I would like to bundle those packages (and any prerequisites) with my tool making the underlying process invisible to most users.

#### Describe the solution you'd like

A scoop configuration that allows you to point Scoop to an install path and it will install all packages there instead, along with any environment set up scripts.

This folder could then be sent along with an application.

#### Describe alternatives you've considered

I've considered purging my Scoop folders, reinstalling, and then bundling the install onto a new computer. This would make updating very challenging and verifying any updates equally challenging. And hard to automate.

**Comment by rashil2000** [comment]
> A scoop configuration that allows you to point Scoop to an install path and it will install all packages there instead, along with any environment set up scripts.

You can point a Scoop installation to any folder you want, by using the SCOOP environment variable. See https://github.com/ScoopInstaller/Install/#advanced-installation for more details.
